internal/common/pagination/pagination_test.go

```diff
@@ -130,6 +130,18 @@ func TestCreatePagination(t *testing.T) {
 	}
 }
 
+// test the requested size is 1 more than then page size.
+func TestCreatePaginationWithoutTotal(t *testing.T) {
+	c := qt.New(t)
+	pPage := utils.IntToPointer(0)
+	pSize := utils.IntToPointer(10)
+	page, size, pag := pagination.CreatePaginationWithoutTotal(pSize, pPage)
```

Comment:
`page` is the offset? 
`size` is the number of returned elements? Why is it the page size + 1? 
And `pag` if the filter, maybe the name could be changed.